scala-thoughts.md

```diff
@@ -0,0 +1,7 @@
+Overall, I'm pretty satisfied with Scala. After using it for a while, it reminds me most of Haskell. While I liked using Haskell in PLs, I've found Scala to be more flexible, usable, and readable in comparison. Some of this is likely due to Haskell only being functional, while Scala can do functional and object-oriented programming. When working on a project, I tend to like that additional flexibility so I can always suit my needs. 
```

Comment:
Maybe thats just me but it feels so much better then haskell. Haskell was very counterintuitive in its syntax, while scala offers alot of same functions but more similar to the main 3 (python, C++ java)